packages/field-base/src/clear-button-mixin.js

```diff
@@ -51,6 +51,8 @@ export const ClearButtonMixin = (superclass) =>
       super.ready();
 
       if (this.clearElement) {
+        this.clearElement.addEventListener('mousedown', (event) => this._onClearButtonAction(event));
```

Comment:
It seems calling `preventDefault()` on `mousedown` / `mouseup` events doesn't prevent the subsequent click event. If so, it would be enough to only focus the input element on `mousedown` while leaving the actual clearing to the click handler. The same probably goes for `touchstart`.

Reply:
Tested this and preventing `mousedown` event does prevent `click` event.

Reply:
Hmm, it doesn't prevent for me:

```
<div class="button">Button</div>

<script>
  const button = document.querySelector('.button');
  button.addEventListener('click', () => {
    console.log('click');
  })
  button.addEventListener('mousedown', (event) => {
    event.preventDefault();
    console.log('mousedown');
  });
</script>
```